pep-0458.txt

```diff
@@ -369,6 +369,12 @@ generate cryptographic keys, and manage a TUF repository.
 
 __ https://github.com/theupdateframework/tuf/blob/v0.11.1/docs/TUTORIAL.md#how-to-create-and-modify-a-tuf-repository
 
+In standard operation the *bin-n* metadata will be created and signed as new
+distributions are uploaded to PyPI. However, there will also need to be a
+one-time online initialization mechanism which to create and sign *bin-n*
+metadata for all existing distributions that are part of the PyPI repository at
```

Comment:
```suggestion
metadata for all existing distributions that are part of the PyPI repository
```

Reply:
Addressed in rebased & force-pushed patches.